04/dbernhard/for_cond_fun2.oat

```diff
@@ -0,0 +1,24 @@
+global a = 0;
+global b = 0;
+
+int program(int argc, string[] args) {
+    b = 5;
+
+    for (; con(b); a = a + 1;) {
+        print_string(string_of_int(a));
+        print_string(", ");    
+    }
+
+    print_int(a);
+    print_string(", b:");
+    print_int(b);
+    
+    return 0;
+}
+
+/* should have no effect */
```

Comment:
I think this comment doesn't apply here, as the return value of the function is actually used